autoreact/ktp_xarray/xarray_wrappers.py

```diff
@@ -18,75 +17,106 @@ def from_data(temps, press, rates):
 
 # Getters
 def get_pressures(ktp):
-    """
-    Gets the pressure values
-    """
+    """Gets the pressure values"""
 
     return ktp.pres.data
 
 
 def get_temperatures(ktp):
-    """
-    Gets the temperature values
-    """
+    """Gets the temperature values"""
 
     return ktp.temp.data
 
 
 def get_values(ktp):
-    """
-    Gets the KTP values
-    """
+    """Gets the KTP values"""
 
     return ktp.values
 
 
 def get_pslice(ktp, ip):
-    """
-    Get a slice at a selected pressure value
-    """
+    """Get a slice at a selected pressure value"""
 
     return ktp.sel(pres=ip)
 
 
 def get_tslice(ktp, it):
-    """
-    Get a slice at a selected temperature value
-    """
+    """Get a slice at a selected temperature value"""
 
     return ktp.sel(temp=it)
 
 
 def get_spec_vals(ktp, it, ip):
-    """
-    Get a specific value at a selected temperature and pressure value
-    """
+    """Get a specific value at a selected temperature and pressure value"""
 
     return ktp.sel(temp=it, pres=ip)
 
 
 def get_ipslice(ktp, ip):
-    """
-    Get a slice at a selected pressure index
-    """
+    """Get a slice at a selected pressure index"""
 
     return ktp.isel(pres=ip)
 
 
 def get_itslice(ktp, it):
-    """
-    Get a slice at a selected temperature index
-    """
+    """Get a slice at a selected temperature index"""
 
     return ktp.isel(temp=it)
 
 
 
 # Setters
 def set_rates(ktp, rates, pres, temp):
-    """
-    Sets the KTP values
-    """
+    """Sets the KTP values"""
 
     ktp.loc[{"pres": pres, "temp": temp}] = rates
     return ktp
+
+
+# Translators
+
+def dict_from_xarray(xarray_in):
```

Comment:
I would change this function signature to the following:
```
def dict_(ktp)
```
since `xarray_in` is just a `ktp` like all the other functions, and `from_xarray` is implied (since all of the functions act on a `ktp` xarrays).

Reply:
(I *would* add the `from_x` qualifier if, for whatever reason, I need to include a function here that didn't act on a `ktp`)